curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-092.md

```diff
@@ -1,20 +1,28 @@
 ---
-id: 5f475e1c7f71a61d913836c6
+id: 5f47fe7e31980053a8d4403b
 title: Part 92
 challengeType: 0
 dashedName: part-92
 ---
 
 # --description--
 
-Add one last image under the `Desserts` heading using the url `https://tinyurl.com/cafe-pie-fcc`. Give the image an `alt` value of `pie icon`.
+It would be nice if the vertical space between the `h2` elements and their associated icons was smaller. The `h2` elements have default top and bottom margin space, so you could change the bottom margin of the `h2` elements to say `0` or another number.
+
+There is an easier way, simply add a negative top margin to the `img` elements to pull them up from their current positions. Negative values are created using a `-` in front of the value. To complete this project, go ahead and use a negative margin of `25px` in the `img` type selector.
```

Comment:
```suggestion
There is an easier way, simply add a negative top margin to the `img` elements to pull them up from their current positions. Negative values are created using a `-` in front of the value. To complete this project, go ahead and use a negative top margin of `25px` in the `img` type selector.
```
To avoid confusion